telegramium-high/src/main/scala/telegramium/bots/high/WebhookBot.scala

```diff
@@ -244,26 +227,51 @@ object WebhookBot {
   def compose[F[_]: Async](
     bots: List[WebhookBot[F]],
     port: Int,
-    host: String = org.http4s.server.defaults.IPv4Host
+    host: String = "0.0.0.0",
+    keystorePath: Option[String] = None,
+    keystorePassword: Option[String] = None
   ): Resource[F, Server] = {
 
-    val setWebhooksResource: Resource[F, Unit] = bots.foldMapM(_.setWebhookResource())
-    val httpRoutes: HttpRoutes[F]              = bots.foldMapK(_.routes())
-
-    val serverResource: Resource[F, Server] =
-      BlazeServerBuilder[F]
-        .bindHttp(port, host)
-        .withHttpApp(httpRoutes.orNotFound)
-        .withServiceErrorHandler { req =>
-          {
-            case e @ DecodingError(message) =>
-              inDefaultServiceErrorHandler(Monad[F])(req)(ResponseDecodingError.default(message, e.some))
-            case throwable => inDefaultServiceErrorHandler(Monad[F])(req)(throwable)
+    val sslContext: Option[Resource[F, SSLContext]] = for {
+      path     <- keystorePath
+      password <- keystorePassword
+    } yield createSSLContext(path, password)
+
+    for {
+      _ <- bots.foldMapM(_.setWebhookResource())
```

Comment:
I think we should call `setWebhook` only after the server is started, Telegram can start sending updates immediately, and if the server is unavailable, the updates will be paused for a while